ibis/backends/bigquery/udf/__init__.py

```diff
@@ -0,0 +1 @@
+from .api import udf  # noqa: F401
```

Comment:
Can you just rename `api.py` to `__init__.py` instead?

Reply:
done!